testing/test_session.py

```diff
@@ -323,7 +323,11 @@ def test_one():
 
     result = testdir.runpytest("--rootdir={}".format(path))
     result.stdout.fnmatch_lines(
-        ["*rootdir: {}/root, inifile:*".format(testdir.tmpdir), "*1 passed*"]
+        [
+            "*rootdir: {}/root, inifile:*".format(testdir.tmpdir),
+            "root/test_rootdir_option_arg.py *",
```

Comment:
Before the fix this was `root/test_rootdir_option_argX/test_rootdir_option_arg.py`.